core/runtime/tests/misc.rs

```diff
@@ -1004,11 +1004,7 @@ async fn test_dynamic_import_module_error_stack() {
 
 #[tokio::test]
 #[should_panic(
-  expected = r#"Failed to initialize a JsRuntime: Top-level await is not allowed in extensions
-
-Caused by:
-    Top-level await promise never resolved
-        at mod:tla:3:11"#
```

Comment:
It's sad to lose this stack trace here :(

Reply:
yeah i think we could add an api in v8 to bring this back. but the way of calling evaluate and then seeing if it resolves immediately is unfortunately not usable since we don't want side effects in error cases.

Reply:
That's reasonable